front/src/components/Search/Filters/GenreSelector.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import GenreSelectionList from '../../GenreSelectionList';
+import { Accordion, ScrollArea, TextInput } from '@mantine/core';
+import { IconSearch } from '@tabler/icons-react';
+import GigzAccordion from '../../GigzAccordion';
+import { useTranslation } from 'react-i18next';
+
+interface IGenreSelectorProps {
+  form: any;
+}
+
+const GenreSelector: React.FC<IGenreSelectorProps> = ({ form }) => {
+  const { t } = useTranslation();
+
+  const [selectedGenre, setSelectedGenre] = useState(form.values.genres ?? []);
+  const [genreFilter, setGenreFilter] = useState('');
+
+  useEffect(() => {
+    form.values.genres = selectedGenre;
+  }, [selectedGenre]);
+
+  const handleAddGenre = (id: number) => {
+    if (!selectedGenre.includes(id)) {
+      setSelectedGenre((old: number[]) => [...old, id]);
+    } else {
+      setSelectedGenre((old: number[]) =>
+        old.filter((idGenre) => idGenre != id)
```

Comment:
not "!==" but "!=" (I have done the same thing in my genre step :disappointed: )